include/z64player.h

```diff
@@ -473,44 +474,50 @@ typedef enum PlayerModelGroup {
     /* 15 */ PLAYER_MODELGROUP_MAX
 } PlayerModelGroup;
 
-typedef enum PlayerEyeIndex {
+typedef struct PlayerFaceIndices {
+    /* 0x0 */ u8 eyeIndex;
+    /* 0x1 */ u8 mouthIndex;
+} PlayerFaceIndices; // size = 0x2
+
+typedef enum PlayerEyes {
```

Comment:
In OoT, they also opted to not use a macro to access the eyes and mouth data from animations i.e. they don’t use `GET_EYE_INDEX_FROM_JOINT_TABLE`, `GET_MOUTH_INDEX_FROM_JOINT_TABLE`, and `GET_APPEARANCE_FROM_JOINT_TABLE`. Just noting that here.